src/engine/strat_engine/backstore/metadata.rs

```diff
@@ -196,68 +233,76 @@ impl StaticHeader {
     }
 
     /// Try to find a valid StaticHeader on a device.
-    /// If there is no StaticHeader on the device, return None.
-    /// If there is a problem reading a header, return an error.
+    /// We will return the first copy that validates as a Stratis BDA, however we will verify both
+    /// copies and if one validates but one does not, we will re-write the one that is incorrect.
+    /// Return None if the static header's magic does not match for *both* copies.
```

Comment:
Suggest cutting the 3 "we will"s in this comment block, I don't think they're needed.

Reply:
Done, thank you.